components/filter/filter.js

```diff
@@ -1026,9 +1028,14 @@ class Filter extends FocusMixin(LocalizeCoreElement(RtlMixin(LitElement))) {
 			switch (dimension.type) {
 				case 'd2l-filter-dimension-set': {
 					dimension.values.forEach(value => {
-						if (value.selected) {
+						if (value.selected && !value.inactive) {
 							const keyObject = { dimension: dimension.key, value: value.key };
-							const text = dimension.valueOnlyActiveFilterText ? value.text : `${dimension.text}: ${value.text}`;
+							let text;
+							if (value.descriptiveText) {
+								text = dimension.valueOnlyActiveFilterText ? value.descriptiveText : `${dimension.text}: ${value.descriptiveText}`;
+							} else {
+								text = dimension.valueOnlyActiveFilterText ? value.text : `${dimension.text}: ${value.text}`;
+							}
```

Comment:
The `SubscriberRegistryController` for `active-filters` is currently only used by `filter-tags`, so this change wouldn't impact anywhere else and also these new properties are only used by the new date-range-value filter, but I still tried to keep this not filter-tags specific.

Let me know any alternative ideas.